pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -0,0 +1,142 @@
+// +build linux
+
+package connections
+
+import (
+	"k8s.io/klog"
+
+	"github.com/ti-mo/conntrack"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+)
+
+var _ ConnTrackPoller = new(connTrackPoller)
+
+type connTrackPoller struct {
+	nodeConfig *config.NodeConfig
+	conntrack  ConnTrack
+}
+
+func NewConnTrackPoller(nodeConfig *config.NodeConfig, conntrack ConnTrack) *connTrackPoller {
+	return &connTrackPoller{
+		nodeConfig,
+		conntrack,
+	}
+}
+
+func (cp *connTrackPoller) DumpFlows(zoneFilter uint16) ([]*flowexporter.Connection, error) {
+	// Get netlink Connection to netfilter
+	err := cp.conntrack.Dial()
+	if err != nil {
+		klog.Errorf("Error when getting netlink conn: %v", err)
+		return nil, err
+	}
+
+	// ZoneID filter is not supported currently in tl-mo/conntrack library.
+	// Dump all flows in the conntrack table for now.
+	// TODO: Open an issue/PR in the library to support zoneID filter.
```

Comment:
I think there is no point in checking-in such a TODO: just open the issue and replace this TODO comment with a link to the issue

Reply:
I wanted to have a PR before opening the issue. Ok with having link here.
Added the link to the issue here.